draft-ietf-rats-architecture.md

```diff
@@ -1013,6 +1013,46 @@ an Attestation Result for any other purpose.
 
 # Security Considerations
 
+## Attester and Attestation Key Protection
+
+(this is a crude first pass to see what interest there is in this)
+
+Implementors need to pay close attention to the isolation and protection of the Attester and the factory processes for provisioning the Attestation Key Material. When these are compromised, the attestation becomes worthless because the attacker can forge attestations.
+
+Attestation applies to use cases with a range of security requirements, so the protections discussed here range from low to high security where low security  may be only application or process isolation by the devices operating system and high security involves specialized HW to defend against physical attacks on a chip.
+
+### On-Device Attester and Key Protection
+
+It is assumed that the Attester is an isolated computing environment like a process, a dedicated chip a TEE or such that collects the Claims, formats them and signs them with the Attestation Key. The Attester must be protected from modification to be sure it behaves correctly. There must also be confidentiality so that the signing key is not captured and used elsewhere to forge attestations.
+
+In many cases the user or owner of the device must not be able to modify or inspect the Attester. For example the owner or user of a mobile phone of FIDO authenticator is not trusted. The point of the attestation is for the relying party to be able to trust the device even though they don’t trust the user or owner. 
```

Comment:
If find this sentence a bit confusing to read:

Please highlight whyt a user or owner must not be able to modify or inspect an Attester. Do you mean effectively the Attesting Environment of the Attester? That would make way more sense to me.

Why would you want a user/owner of a phone to provide biometric digests, pins or passwords, if you do not trust her in the first place?

"The point of the attestation": I really do not know what that means here. Please remove 'attestation' and elaborate on the goal of evidence generation earlier in the text please. Alternatively, "remote attestation procedure" might already be good enough?